bitnami/fluent-bit/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.rbac.create -}}
+apiVersion: {{ include "common.capabilities.rbac.apiVersion" . }}
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/part-of: fluent-bit
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ include "fluentBit.fullname" . }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "fluentBit.serviceAccountName" . }}
+    namespace: {{ .Release.Namespace }}
```

Comment:
Use common.names.namespace just like it is above

Reply:
Fixed [deb4b4a](https://github.com/bitnami/charts/pull/15453/commits/deb4b4ac3e74a7576dc91b4e84a6fbd2f0eb8b02)